vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/tabs/TabsWrapTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+
+package com.vaadin.flow.component.tabs;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.vaadin.flow.router.RouteConfiguration;
+import com.vaadin.testbench.unit.UIUnitTest;
+import com.vaadin.testbench.unit.ViewPackages;
+
+@ViewPackages
+class TabsWrapTest extends UIUnitTest {
+
+    TabsView view;
+    TabsWrap<Tabs> tabs_;
+
+    @BeforeEach
+    void init() {
+        RouteConfiguration.forApplicationScope()
+                .setAnnotatedRoute(TabsView.class);
+        view = navigate(TabsView.class);
+        tabs_ = wrap(view.tabs);
+    }
+
+    @Test
+    void select_notUsable_throws() {
```

Comment:
Should notUsable also assert with index the same way the disable and notExist does?

Reply:
done